cli/testflinger_cli/__init__.py

```diff
@@ -119,6 +125,19 @@ def _print_queue_message():
     )
 
 
+def make_relative(path: Path) -> Path:
+    """Return resolved relative `path`"""
+    if path.is_absolute():
+        # strip leading '/' from absolute path
+        path = path.relative_to(path.root)
+    # resolve the path and make relative again
+    return path.resolve().relative_to(Path.cwd())
+
+
+class AttachmentError(Exception):
+    """Exception thrown when attachments fail to be submitted"""
```

Comment:
Is it supposed to have an empty body?

Reply:
Yes, the linter doesn't like `pass` along with the docstring (which is also considered a statement essentially).

Reply:
I actually didn't mean to “pass” keywork but the class implementation itself😅